spec/controllers/users_controller_spec.rb

```diff
@@ -55,8 +55,8 @@
       @params[:id] = user.id
     end
 
-    it_should_allow_admin_only do
-      expect(assigns[:user]).to eq(user)
+    context "when create_users is active", feature_setting: { create_users: true } do
```

Comment:
This seems reasonable to me, but I recall someone mentioning that with feature-flags like this, we prefer to test both contexts (with the feature on and the feature off).

What makes this case different?